client/src/context/AuthContext.tsx

```diff
@@ -11,76 +11,160 @@ import {
 } from 'firebase/auth';
 import Cookies from 'js-cookie';
 
+const { auth } = app;
 const csrftoken = Cookies.get('csrftoken');
 
 export interface User {
   username: string, 
   email: string, 
-  uuid: string | unknown,
-  location: number | unknown
+  uuid: string | null,
+  location: string
```

Comment:
Location will always revert back to "13" for Tokyo if the user is not logged in or hasn't been onboarded yet, therefore, it will always be a string.